exporter/importables.go

```diff
@@ -35,6 +37,49 @@ var (
 	globalWorkspaceConfName = "global_workspace_conf"
 )
 
+type sqlaListResponse struct {
+	Results    []map[string]interface{} `json:"results"`
+	Page       int64                    `json:"page"`
+	TotalCount int64                    `json:"count"`
+	PageSize   int64                    `json:"page_size"`
+}
+
+// Generic function to list objects related to the SQL Analytics
```

Comment:
DBSQL

Reply:
will fix - it was started when it was SQL Analytics :-)